helpers/getImageSrcset.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+const utils = require('handlebars-utils');
+const SafeString = require('handlebars').SafeString;
+const common = require('./lib/common');
+
+const factory = () => {
+    return function(image, defaultImageUrl) {
+        // Regex to test size string is of the form 123x123 or 100w
+        const sizeRegex = /(^\d+w$)|(^(\d+?)x(\d+?)$)/;
+        // Regex to test to that srcset descriptor is of the form 1x 1.5x 2x OR 123w
+        const descriptorRegex = /(^\d+w$)|(^([0-9](\.[0-9]+)?)x)$/;
+
+        const options = arguments[arguments.length - 1];
+
+        if (utils.isUndefined(defaultImageUrl)) {
+            defaultImageUrl = '';
+        }
+
+        if (!utils.isObject(image) || !utils.isString (image.data)
+            || !common.isValidURL(image.data) || image.data.indexOf('{:size}') === -1) {
```

Comment:
Well technically size should not be a requirement. It could be a theme asset where the theme developer already has  all the versions of the asset inside the theme. Something like this ```<img srcset="elva-fairy-320w.jpg 320w,
             elva-fairy-480w.jpg 480w,
             elva-fairy-800w.jpg 800w"
     src="elva-fairy-800w.jpg" alt="Elva dressed as a fairy">```

Reply:
You don't need this helper then - this is for sizing the same images that are a candidate for `{{getImage}}`